tonic/src/transport/mod.rs

```diff
@@ -129,7 +129,3 @@ pub use self::server::ServerTlsConfig;
 #[cfg(feature = "tls")]
 #[cfg_attr(docsrs, doc(cfg(feature = "tls")))]
 pub use self::tls::Identity;
-
-#[cfg(feature = "channel")]
```

Comment:
I don't understand how your commit message describes this change? It seems to remove this instance and makes another instance private, but this commit doesn't "move" anything in the sense of creating something new.

Reply:
The definition of BoxFuture used in the implementation of the channel feature is moved to the channel module.